portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,10 +35,99 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+   // private List<String> messages;
```

Comment:
For what it's worth I would get rid of the commented-out code. It'll be in your GitHub history if you need it!